packages/thirdweb/src/exports/auth.ts

```diff
@@ -6,6 +6,10 @@ export {
   verifyContractWalletSignature,
   type VerifyContractWalletSignatureParams,
 } from "../auth/verify-signature.js";
+export {
+  type VerifyTypedDataParams,
+  verifyTypedData,
+} from "../auth/verify-typed-data.js";
```

Comment:
should this be exported from `auth` or from utils or something? What does it have to do with `SIWE` auth